src/style.css

```
@@ -1 +1,80 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  background-color: #fff;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.list {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+}
+
+.__link {
+  display: block;
+  margin: 0;
+  height: 40px;
+}
+
+.nav__list {
+  margin: 0 5px;
+  padding: 0 5px;
+}
```

```suggestion
.nav__item {
  margin: 0 5px;
  padding: 0 5px;
}
```
it is an item, not a list 😉
and your 1st and last elements don't need margins (left and right respectively) 🙃